uint8 core_0
uint8 core_1
uint8 core_2
uint8 core_3
uint8 ssd
